### PR TITLE
Fix lint error in broker server.

### PR DESCRIPTION
**Pull request by bmelville**
Signed-off-by: Brendan Melville <bmelville@google.com>

### CODE REVIEW

**Comment by duglin** [comment]
LGTM

File: contrib/broker/server/server.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ func CreateHandler(c controller.Controller) http.Handler {
 		controller: c,
 	}
 
-	var router *mux.Router = mux.NewRouter()
+	var router = mux.NewRouter()
```
Comment (arschles) [inline contrib/broker/server/server.go]: this would be better written `router := mux.NewRouter()`

**Comment by arschles** [comment]
LGTM

File: contrib/broker/server/server.go (hunk 1 above)
Comment (pmorie) [inline contrib/broker/server/server.go]: agree, will merge anyway.

**Comment by pmorie** [comment]
LGTM, merging.

**Pull request merged**